anipra-walnutcreek.md

```diff
@@ -0,0 +1,10 @@
+# San Francisco, CA
+
+### Restaurants
+
+### Local Attractions
+
+- Fisherman's wharf
```

Comment:
I was there when I was a teenager. I loved it. Are you from San Francisco originally?